File: Beecrowd/1115 Quadrante/main.py
X = 1
Y = 1
while X != 0 and Y != 0:
  entrada = [int(i) for i in input().split()]
  X = entrada[0]
  Y = entrada[1]
  if X != 0 and Y !=0:
    if X < 0  and Y < 0:
      print("terceiro")
    elif X < 0 and Y > 0:
      print("segundo")
    elif Y < 0 and X > 0:
      print("quarto")
    else:
      print("primeiro")

File: Beecrowd/1115 Quadrante/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int X, Y;
	
	do{

		cin >> X;
		cin >> Y;
	
		if(X != 0 && Y !=0){

			if(X < 0  && Y < 0)
				cout << "terceiro\n";
			else if(X < 0 && Y > 0)
				cout << "segundo\n";
			else if(Y < 0 && X > 0)
				cout << "quarto\n";
			else
				cout << "primeiro\n";
		}
	}while (X != 0 && Y != 0);
 
    return 0;
}